python/ray/rllib/a3c/a3c.py

```diff
@@ -154,7 +147,8 @@ def _restore(self, checkpoint_path):
         ])
         self.local_evaluator.restore(extra_data["local_state"])
 
-    def compute_action(self, observation):
+    def compute_action(self, observation, state=[]):
```

Comment:
better to avoid using mutable objects as default values, perhaps

`state=None` and `state = [] if state is None else state`

Reply:
Done